src/components/Slider/Slider.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from "react";
+import "./Slider.css";
+import ItemsCarousel from "react-items-carousel";
+import SliderCard from "../SliderCard/SliderCard.js";
```

Comment:
`SliderCard.js`: the extension is not necessary here.
I would also recommend to group imports from libraries together, only then have the local imports (`../something`).